languages/en.ini

```diff
@@ -1089,6 +1089,8 @@ Photo = "Photo"
 Physical Description = "Physical Description"
 Physical Object = "Physical Object"
 pick_up_location = "Pickup Location"
+pick_up_location_label_delivery = "Delivery Address"
```

Comment:
Since translation is already underway for release 10.1, we have a freeze on adding new translation strings. Since these are essentially just examples, if you really want to get this included in the release, perhaps we could remove the example configurations and language strings from this PR and introduce them in a separate PR targeted for 11.0.

Reply:
Ok, I've taken that advice.  I did need to leave in one new config, `allowDelivery = false`, to effectively disable to code, because it breaks without some of the config (which itself breaks without translation).  The second PR is #3930.